README.md

```diff
@@ -25,28 +82,218 @@ This repository contains extension code for SQL Server input and output bindings
 1. Install the extension.
 
     ```bash
-    dotnet add package Microsoft.Azure.WebJobs.Extensions.Sql --version 1.0.0-preview1
+    dotnet add package Microsoft.Azure.WebJobs.Extensions.Sql --version 1.0.0-preview2
+    ```
+
+1. Ensure you have Azure Storage Emulator running. For information on the Azure Storage Emulator, refer [here](https://docs.microsoft.com/en-us/azure/storage/common/storage-use-emulator#get-the-storage-emulator)
+
+1. Set the SqlConnectionString. Your connection string can be found in your SQL database resource by going to the left blade and clicking 'Connection strings'. Copy the Connection String to 'local.settings.json', so '"SqlConnectionString": [Connection String]' is in 'Values:'.
```

Comment:
I would include what their final host.json needs to look like (like what you did for AzureWebJobsDashboard, except show the full host.json w/ both settings required)

Reply:
That makes sense. I'll make the update

Reply:
Let me know if the update is what you were thinking